element114/tusari/_posts/1992-10-01-vertical-parallax-fld.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 1992-10-01 20
+title:  "Vertical parallax FLD"
+csdb: 178390
+coders:
+  - name: "Martin Piper"
+    csdb: 19341
+record: 4
+size: 320x200
+fps: 50
+tags: fld fld-parallax fld-parallax-vertical
+---
+![Vertical parallax FLD](/c64wrd/element114/tusari/vertical-parallax-fld.png)
```

Comment:
I think we should put it into a directory /game/tusari instead of /element114/tusari, because if there will be multiple records from games then there will be a lot of almost empty subdirectories

Reply:
Also done